robottelo/vm.py

```diff
@@ -326,6 +326,25 @@ def configure_puppet(self, rhel_repo=None):
         # stage can be ignored.
         self.run(u'puppet agent -t 2> /dev/null')
 
+    def execute_foreman_scap_client(self, policy_id=None):
+        """Executes foreman_scap_client on the vm/clients to create security
+        audit report.
+
+        :param policy_id: The Id of the OSCAP policy.
+        :return: None.
+
+        """
+        if policy_id is None:
+            result = self.run(
+                u'awk -F "/" \'/download_path/ {print $4}\' '
+                '/etc/foreman_scap_client/config.yaml'
+            )
+            policy_id = result.stdout[0]
+        self.run(u'foreman_scap_client {0}'.format(policy_id))
+        if result.return_code != 0:
+            raise VirtualMachineError(
+                'Failed to execute foreman_scap_client run.')
```

Comment:
> I think you are referring to policy_id when you say OSCAP ID.

Yup, that's right. I figured OSCAP ID == `policy_id` after reading `The Id of the OSCAP policy.`.

> It's not required, as the policy_id will have no significance while troubleshooting it, as it's related to functional stuff not automation stuff.

Hunh. I don't understand the distinction between functional and automation.

Is there anything that would be useful? Maybe the contents of `result.stderr`?